### DISCUSSION TITLE
TV Vest Rogaland

### DISCUSSION THREAD

**Discussion by ghost** [iptv-org/community#351] [Show and tell]
### Channel Name

TV Vest Rogaland

### Channel ID

TVVestRogaland.no

### Website

https://video.tvvest.no/se-tv-vest-rogaland-direkte-uten-abonnement

### Stream URL

https://live01.delivery.twentythree.com/live/amlst:u7lt3cvm1s5cr9uojqjj7kbt27:11596818-d53ea5a866639438de94.mp4/chunklist_b928000.m3u8

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

**Comment by BellezaEmporium** [comment]
Seems the middle part changes over time.

**Comment by BellezaEmporium** [comment]
It's now https://live01.delivery.twentythree.com/live/amlst:26suq6ljlrctrp36brv6ct2kkf:11596818-d53ea5a866639438de94.mp4/playlist.m3u8

**Comment by freearhey** [comment]
https://live01.delivery.twentythree.com/live/amlst:ohja91i2e760itd4q8nkhqqi3k:11596818-d53ea5a866639438de94.mp4/playlist.m3u8
<img width="1413" alt="image" src="https://user-images.githubusercontent.com/7253922/190888840-a7a54bb8-64fc-437f-8347-4c544394b370.png">

**Comment by Alstruit** [comment]
Looks like the link changes and expires some time.

**Comment by freearhey** [comment]
https://live01.delivery.twentythree.com/live/amlst:u7lt3cvm1s5cr9uojqjj7kbt27:11596818-d53ea5a866639438de94.mp4/chunklist_b928000.m3u8 (offline)
<img width="831" alt="image" src="https://user-images.githubusercontent.com/7253922/232229271-f55235b6-fc28-4bc4-a1d4-9d0bf4a846dc.png">

https://live01.delivery.twentythree.com/live/amlst:26suq6ljlrctrp36brv6ct2kkf:11596818-d53ea5a866639438de94.mp4/playlist.m3u8 (offline)
<img width="832" alt="image" src="https://user-images.githubusercontent.com/7253922/232229313-67e3103d-ebc5-490c-84bc-4178d202447e.png">
